lib/setup/setupInstall.js

```diff
@@ -1285,6 +1285,8 @@ function Install(options) {
                     .filter(id => knownObjIDs.indexOf(id) === -1)
                 ;
                 knownObjIDs.push.apply(knownObjIDs, newObjs);
+                // meta ids can also be present as files
+                metaFilesToDelete.push.apply(metaFilesToDelete, newObjs);
```

Comment:
I find this one pretty weird, but followed the existing style

Reply:
I also wondered and no idea why :-)

Reply:
> I find this one pretty weird

That was probably from a refactor I did a while back to get the old deletion code cleaned up. There's probably a nicer way though.

Reply:
But why not simply push? It is a normal array and we add a value there ?! or is it to allow to do that and also "pass it tou the outside"?

Reply:
Because then it would only be block-scoped.

Reply:
Ahhh... now I know. That is the old (ES5) syntax for `knownObjIDs.push(...newObjs)`. If you just pushed, you would add the entire new array as a single item.